src/main/java/uk/gov/hmcts/darts/log/service/impl/AudioLoggerServiceImpl.java

```diff
@@ -14,8 +15,8 @@ public class AudioLoggerServiceImpl implements AudioLoggerService {
     @Override
     public void audioUploaded(AddAudioMetadataRequest request) {
         log.info("Audio uploaded: courthouse={}, courtroom={}, started_at={}, ended_at={}",
-                 request.getCourthouse(),
-                 request.getCourtroom(),
+                 DataUtil.toUpperCase(request.getCourthouse()),
+                 DataUtil.toUpperCase(request.getCourtroom()),
```

Comment:
I think the `AddAudioMetadataRequest` has already been preProcessed at the controller level, so do we need this transformation here? Belt and braces?

Reply:
This is creating a brand new object so we create it here with the correct values instead of consuming an already created object

Reply:
Agreed :)

I left these here incase we re-use this logging anywhere that is not already covered by a preProcess method in the future as the format is important for DynaTrace it seemed sensible to leave it here too :)

As such I left these here for Belt and braces